bin/start_server.sh

```diff
@@ -24,8 +24,12 @@ fi
 echo "Running migration..."
 
 ./manage.py initialize_assets
```

Comment:
What about initializing assets? Is it atomic?

Reply:
I'd like to see the whole process atomic. 

It's like

```
mkdir data.tmp 
.... do all the stuff
mv data.tmp data
```

So, the whole process is atomic, including other steps, like asset initialization. 

The whole file looks strange. It's called `start_sever.sh`, so I expect it will run all the time, but wouldn't it migrate every time it starts?

Reply:
@renatgalimov,
* `initialize_assets` just makes sure if the required directories are present.
  * Maybe it's safe to remove it, I'll check. See screenshot below (from `start_server.sh`) for details.

![image](https://github.com/user-attachments/assets/ac1da7c6-a921-4988-86ad-b62b2a042131)

* Yes, even a long time ago, the migration runs every time the server starts. Is that an acceptable behavior?